Makefile

```diff
@@ -7,6 +7,7 @@ fmt:
 	goimports -w main.go
 
 test:
+	API_KEY=b04ad8db6f75cbd1a02e6e4c8e1e1272 \
```

Comment:
you don't need to provide real api key in your tests. Please remove it and i'll add one more comment below: i'll describe you a workaround for that :)